packages/dd-trace/src/telemetry/dependencies.js

```diff
@@ -5,10 +5,11 @@ const parse = require('module-details-from-path')
 const requirePackageJson = require('../require-package-json')
 const { sendData } = require('./send-data')
 const dc = require('diagnostics_channel')
+const { fileURLToPath } = require('url')
 
 const savedDependencies = []
 const detectedDependencyNames = new Set()
-const FILE_PATH_START = `file:${path.sep}${path.sep}`
+const FILE_PATH_START = `file://`
```

Comment:
`FILE_PATH_START` or `FILE_URI_START`?

Reply:
Second one sounds better to me

Reply:
also to me, because we are talking about URI, to differenciate it from path